### DISCUSSION TITLE
Getting error when setting a non-English lang as source language in an Android Project

### DISCUSSION THREAD

**Discussion by Alireza-Farahani** [WeblateOrg/weblate#6129] [Q&A]
We are a Iranian team working on an Android Project. String resources are defined in Persian and should be translated to other language later.

English translations are in `values/strings.xml` and Persian ones in `values-fa/strings.xml`. On defining our project Weblate componen, when I set "source language" to Persian and "base multi lingual file" to `values/strings.xml`, I get this error
```
There is more than one file for Persian language, please adjust the filemask and use components for translating different resources
```
I moved Persian translations to `values.xml` and English ones to `values-en`, but no luck; got same error. When I set source language as English, component is created without problem.

What am I doing wrong? Is it possible to set a non-English language as source language in an Android app?

---

Weblate version: `4.5.1`

**Comment by nijel** [comment]
I thought that `values/strings.xml` is expected to contain English strings in Android. In that case, using [secondary languages](https://docs.weblate.org/en/latest/user/profile.html#secondary-languages) to display Persian next to English might be a solution for you.

Weblate will complain if `values-fa/strings.xml` is present, and Persian is the source language with `values/strings.xml` as base file - because in this case, there are two files for Persian.

**Reply by Alireza-Farahani** [reply]
I see. secondary language makes it better but it's a user preference not a component setting. 
So I **have to** set base multilingual file to `values/strings.xml` and source language to English, right? (I also tried to set the base file to `values-fa/strings.xml` and source language to Persian, but after component is created, it complains about component not configured correctly, check out file mask ... )
